.github/FUNDING.yml

```diff
@@ -0,0 +1,2 @@
+github: contao-association
+custom: "https://to.contao.org/supporter"
```

Comment:
I guess this needs to be changed to?

```suggestion
custom: https://to.contao.org/supporter
```

Reply:
According to https://docs.github.com/en/github/administering-a-repository/displaying-a-sponsor-button-in-your-repository

> Note: If a custom URL in an array includes :, you must wrap the URL in quotes. For example, "https://www.paypal.me/octocat".

Reply:
The code style checker doesn’t seem to like that, maybe try single quotes?